CentralManagement/CentralManagementSetupWeb.md

```diff
@@ -247,6 +248,32 @@ You should received a notification similar to this:
 
 ![Test email sent successfully](images/features/ccm/test_email_sent_correctly.png)
 
+### Step 4.3: LDAP Configuration
+

```

Comment:
It's probably worth a revisit to mention this is an optional configuration step.